Pages/BluetoothPage.xaml

```diff
@@ -61,6 +61,7 @@
                       ItemsSource="{Binding bleDeviceList}"
                       IsVisible="{Binding usingBluetooth}"
                       ItemSelected="listView_ItemSelected"
+                      VerticalScrollBarVisibility="Never"
```

Comment:
I have had to scroll to find my desired serial number.

Reply:
You can still scroll, this just gets rid of the bar which is an eyesore on mobile